.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: Auth0 Community
-    url: https://community.auth0.com/c/sdks/5
```

Comment:
The path `/c/sdks/5` no longer exists.

Reply:
Should we use https://community.auth0.com/tag/auth0-oidc-client-net instead?

Reply:
Given that the forums include a [FAQ](https://community.auth0.com/c/faq/42) section, it'd be wise to encourage developers do a site-wide search, as their problem might not be specific to this SDK, but could be answered in one of those FAQs.